pandas/tests/io/parser/header.py

```diff
@@ -360,3 +361,39 @@ def test_mangles_multi_index(self):
                                   ('A', 'one.1.1'), ('B', 'two'),
                                   ('B', 'two.1')]))
         tm.assert_frame_equal(df, expected)
+
+    @pytest.mark.parametrize("index_col", [None, [0]])
+    @pytest.mark.parametrize("columns", [None,
+                                         (["", "Unnamed"]),
+                                         (["Unnamed", ""]),
+                                         (["Unnamed", "NotUnnamed"])])
+    def test_multi_index_unnamed(self, index_col, columns):
+        header = [0, 1]
```

Comment:
can you add the issue/PR number here and a bit of explaination of what you are testing.

Reply:
Done.